src/types/dsl.model.ts

```diff
@@ -49,6 +51,7 @@ export class IDsl
       this.input = JSON.parse(rawDsl.input);
       this.deployable = JSON.parse(rawDsl.deployable);
       this.supportsMultipleFlows = JSON.parse(rawDsl.supportsMultipleFlows);
+      this.supportsResourceDescription = JSON.parse(rawDsl.supportsResourceDescription);
```

Comment:
Could we please add a little test to check this out? mostly about what happens if this line fails and then we get the default value.

Reply:
Not only adding a line in dsl.model.test.ts and test with rawCapabilitiesStub[0]? is there an example, possibly for supportsMultipeFlows?

Reply:
nevermind, found one at the very bottom of the `dsl.model.test.ts`

Reply:
done :+1: